src/outputs/intram_by_civil_defence.py

```diff
@@ -12,46 +11,75 @@ def output_intram_by_civil_defence(
     df: pd.DataFrame,
     config: Dict[str, Any],
     write_csv: Callable,
-    civil_defence_detailed: pd.DataFrame,
-):
+) -> pd.DataFrame:
     """Run the outputs module.
 
     Args:
-        df (pd.DataFrame): The dataset main with weights not applied
+        df (pd.DataFrame): The dataset main.
+        df_for_output (pd.DataFrame): The summed dataset for output
         config (dict): The configuration settings.
         write_csv (Callable): Function to write to a csv file.
-         This will be the hdfs or network version depending on settings.
-        civil_defence_detailed (pd.DataFrame): Detailed schema of C/D output
+            This will be the hdfs or network version depending on settings.
+    Returns:
+        df_for_output (pd.DataFrame): Total intramural expenditure by Civil or Defence
+    """
 
+    df_for_output = generate_intram_by_civil_defence(
+        df)
+
+    # Outputting the CSV file
+    _save_output_intram_civil_def_as_csv(df_for_output, config, write_csv)
+    
+    return df_for_output
```

Comment:
this doesn't need to be returned, and when the function is called in output_main.py we don't expect something to be retuned.

I suggest return None (and update docstring)

Reply:
I have addressed this comment.